mediorum/server/serve_legacy.go

```diff
@@ -36,6 +36,8 @@ func (ss *MediorumServer) serveLegacyCid(c echo.Context) error {
 		return ss.redirectToCid(c, cid)
 	}
 
+	log.Println("serving legacy cid", cid)
```

Comment:
serveBlob looks great, just for the logs in serveLegacyCid+serveLegacyDirCid+redirectToCid could you make sure the logs all have the cid in as a field rather than just in the message?